spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -70,6 +70,15 @@
       expect(response.content_type).to eq('application/json')
       expect(response).to match_response_schema('autocomplete')
     end
+
+    it 'does not return results for extension institutions' do
+      create(:version, :production)
+      create(:institution, :contains_harv, campus_type: 'E')
```

Comment:
`autocomplete` only matches text that _start_ with the search term.  This test is passing because the autocomplete search is failing to find a match, and not necessarily because `campus_type` is `E`.
You can change it like so:
```suggestion
      create(:institution, :start_like_harv, campus_type: 'E')
      create(:institution, :start_like_harv, campus_type: 'Y')
```
And in the expectation below:
`expect(JSON.parse(response.body)['data'].count).to eq(1)`
Since both institution records are identical other than their `campus_type` value, this will properly show that the reason only one result was returned due to `campus_type` value.